<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPort</class>
 <widget class="QMainWindow" name="SerialPort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerialPort</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ClearRecvBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>340</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>清空接收区</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="SendDataEdt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Recv">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>421</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>串口数据接收</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <property name="horizontalSpacing">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QTextEdit" name="RecvDataEdt">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="ReflushSerPortBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>刷新串口</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_SerPortSetting">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>21</y>
      <width>191</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <widget class="QPushButton" name="OnOffBtn">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>270</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>隶书</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
    <widget class="QComboBox" name="FlowCtrlCmb">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Code Pro</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="FlowCtrlLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>53</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>流控制:</string>
     </property>
    </widget>
    <widget class="QLabel" name="SerPortLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>53</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>串口号:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="PortNumCmb">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Code Pro</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="BoaudLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>53</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>波特率:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="baudRCmb">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Code Pro</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="CheckLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>53</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>校验位:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="DPaityCmb">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Code Pro</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="DataLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>53</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>数据位:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="DataBCmb">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Code Pro</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="StopLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>53</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>停止位:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="StopBCmb">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Source Code Pro</family>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="SendBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>470</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>隶书</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>410</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX发送</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>340</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX显示</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>440</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>440</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>重复发送</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>440</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
